Assets/Scripts/Audio/AudioManager/MechanicAudioManager.cs

```diff
@@ -47,15 +48,15 @@ public class MechanicAudioManager : AbstractAudioManager {
     }
 
     public void playButtonSound() {
-        playOneShotSound(buttonClip, button, buttonVolume);
+		playOneShotSound(instance.buttonClip, instance.button, buttonVolume);
```

Comment:
do you need to do instance.bla because you are in the mechanicAudioManager so you can access it's own variables?